task.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
```

Comment:
we need another class for managing the task.

you need to create a TaskManager class it is going to be kinda service layer for our tasks